charts/rasa-x/templates/event-service-deployment.yaml

```diff
@@ -75,10 +75,10 @@ spec:
             scheme: {{ default "HTTP" .Values.eventService.readinessProbe.scheme }}
         {{ end }}
         env:
-        {{- if eq "true" (include "is_enterprise" .) }}
+        {{- if .Values.rasax.disableTelemetry }}
```

Comment:
This part doesn't look right to me -
Shouldn't we completely remove the conditional and set the var to `true`? Since we're always enterprise now, and we enable telemetry by default.

Reply:
If we do that, the users won't be able to opt out of telemetry if needed.

Reply:
[BT-107](https://rasahq.atlassian.net/browse/BT-107) says "The customer should have the option to disable telemetry recording if they wish to do so."

Reply:
Got it, didn't know the ticket and that we're actually changing behaviour in addition to removing the enterprise stuff. In the old code, there was no way to turn off telemetry unless enterprise, right? Since `$.Values.rasa.disableTelemetry` was not evaluated.